hooks/build

```diff
@@ -6,10 +6,10 @@ echo "Build hook running"
 echo "IMAGE_NAME = ${IMAGE_NAME}"
 echo "DOCKER_REPO = ${DOCKER_REPO}"
 echo "CACHE_TAG = ${CACHE_TAG}"
```

Comment:
is cache tag used for anything?

Reply:
It isn't used in the way that it is referenced, but it is embedded in the IMAGE_NAME. Its just informative at this point.